iot-hub/Samples/service/CleanupDevicesSample/CleanupDevicesSample.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Sas;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    public class CleanupDevicesSample
+    {
+        private const string ImportExportDevicesFileName = "devices.txt";
```

Comment:
Is this file created as part of exporting device identities?

Reply:
Yes, it is.
I'll add a link to the doc that describes the process, in our sample: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-bulk-identity-mgmt

Reply:
Cool, this is where I think readme would be useful. Even adding a link to this document and explaining the steps you have in comments would be awesome.

That also brings another question - Do the snippets in this doc need to be updated as well?

Reply:
I've added a link to the doc in the sample.

As for the doc snippets - they've used v11 of the storage sdk, and the snippets in the doc refer to the types available in there. (the ones I used in the sample are from v12). It would be nice to have the doc updated to the latest version of storage sdk, but I am not sure if that is something the owning team wants to do.
If this is something you're interested in and would like to pursue, that'd be great. It would definitely benefit customers to have code snippets that are compatible with the latest version of the sdk.

Reply:
Check with Robin Shahan.